MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -939,7 +963,8 @@ protected override string GetCommandLineArguments(string outputPath, EncodingOpt
             if (isEncoding && state.TargetFramerate > 0)
             {
                 float startTime = 1 / (state.TargetFramerate.Value * 2);
```

Comment:
This line has changed in dev, which might mean that some negative number bugs are fixed